run-workflow.sh

```diff
@@ -1,21 +1,31 @@
 #!/bin/sh
 set -eu
 
-if [ ${#} != 1 ]
+if [ ${#} -lt 1 ]
 then
-  echo "Requires PROCS!"
+  echo "Usage: $0 PROCS [DATASPACES|FLEXPATH]"
   exit 1
 fi
 
 PROCS=$1
+STAGING=FLEXPATH
+
+if [ ${#} -gt 1 ]
+then
+  if [ "$2" = "DATASPACES" ]
+  then
+    rm -f conf srv.lck *bp
+  fi
+  STAGING=$2
+fi
 
 # USER: Set this to the correct location:
-LAUNCH=$HOME/project/mpix_launch_swift/src
+LAUNCH=
 
 MACHINE=${MACHINE:-}
 
 THIS=$( dirname $0 )
 
 rm -f heat.bp_*_info.txt
-stc -p -u -I $LAUNCH -r $LAUNCH workflow.swift
-turbine -n $PROCS $MACHINE workflow.tic
+stc -p -u -I $LAUNCH -r $LAUNCH workflow.swift 
+turbine -n $PROCS $MACHINE workflow.tic -s=${STAGING}
```

Comment:
I know this is already merged, but what does `-s` do here? Tubine help page says we should be providing a settings.sh file here